restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/sipua/SipProfile.java

```diff
@@ -33,11 +33,13 @@
 	private static final String TAG = "SipProfile";
 	private  String localIp;
 	private  int localPort = 5080;
-	private  String transport = "tcp";
+	private  String transport = "udp";
```

Comment:
@mehturt: Any specific rationale for changing to udp? We used to have udp but changed to tcp as its more robust.